client/state/sites/selectors.js

```diff
@@ -23,7 +24,8 @@ import createSelector from 'lib/create-selector';
  * @return {?Object}        Site object
  */
 export function getSite( state, siteId ) {
-	return state.sites.items[ siteId ] || null;
+	const site = state.sites.items[ siteId ];
+	return site ? Object.assign( site, getAttributes( site ) ) : null;
```

Comment:
Should we memoize this selector to avoid computing things every time we getSites (which is a lot)?